blt/urls.py

```diff
@@ -107,7 +112,7 @@
 handler500 = "website.views.handler500"
 
 urlpatterns = [
-    path("captcha/", include("captcha.urls")),
+    # path("captcha/", include("captcha.urls")),
```

Comment:
please keep the existing captcha code so the existing report an issue page can continue to work

Reply:
@AtmegaBuzz please revert this line